setup.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python3
```

Comment:
Would it make sense to not create a setup.py file and create a toml file for this form the get go?

Reply:
Yeah, can migrate it to toml. Although, not sure if we can entirely escape setup.py yet...